ci/docker/runtime_functions.sh

```diff
@@ -542,7 +542,7 @@ build_ubuntu_cpu_clang60_mkldnn() {
     build_ccache_wrappers
 
     make \
-        ENABLE_TESTCOVERAGE=1         \
+        ENABLE_TESTCOVERAGE=0         \
```

Comment:
is this enough, ENABLE_TESTCOVERAGE being used in other places too ?

Reply:
based on what I observe in PR queue, only Clang 6 with MKLDNN failed

Reply:
I end up disabling  the Clang 6 with MKLDNN job to avoid the situation where other task use the same function

Reply:
okay thanks!